datadog/resource_datadog_logs_archive_order.go

```diff
@@ -0,0 +1,91 @@
+package datadog
+
+import (
+	"fmt"
+	datadogV2 "github.com/DataDog/datadog-api-client-go/api/v2/datadog"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"strings"
+)
+
+func resourceDatadogLogsArchiveOrder() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogLogsArchiveOrderCreate,
+		Update: resourceDatadogLogsArchiveOrderUpdate,
+		Read:   resourceDatadogLogsArchiveOrderRead,
+		Delete: resourceDatadogLogsArchiveOrderDelete,
+		Exists: resourceDatadogLogsArchiveOrderExists,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {Type: schema.TypeString, Required: true},
+			"archives": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+		},
+	}
+}
+
+func resourceDatadogLogsArchiveOrderCreate(d *schema.ResourceData, meta interface{}) error {
+	return resourceDatadogLogsArchiveOrderUpdate(d, meta)
```

Comment:
Are you sure we want to do that? Not the same thing as the integration pipeline, where we just support imports?

Reply:
It's the same behavior of endpoint to reorder pipeline